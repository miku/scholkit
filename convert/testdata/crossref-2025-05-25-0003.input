{
  "indexed": {
    "date-parts": [
      [
        2025,
        5,
        25
      ]
    ],
    "date-time": "2025-05-25T10:02:27Z",
    "timestamp": 1748167347288
  },
  "reference-count": 30,
  "publisher": "Elsevier BV",
  "license": [
    {
      "start": {
        "date-parts": [
          [
            2017,
            2,
            1
          ]
        ],
        "date-time": "2017-02-01T00:00:00Z",
        "timestamp": 1485907200000
      },
      "content-version": "tdm",
      "delay-in-days": 0,
      "URL": "https://www.elsevier.com/tdm/userlicense/1.0/"
    }
  ],
  "content-domain": {
    "domain": [
      "gaitposture.com",
      "clinicalkey.jp",
      "clinicalkey.com",
      "clinicalkey.es",
      "clinicalkey.com.au",
      "clinicalkey.fr",
      "elsevier.com",
      "sciencedirect.com"
    ],
    "crossmark-restriction": true
  },
  "short-container-title": [
    "Gait &amp; Posture"
  ],
  "published-print": {
    "date-parts": [
      [
        2017,
        2
      ]
    ]
  },
  "DOI": "10.1016/j.gaitpost.2016.11.040",
  "type": "journal-article",
  "created": {
    "date-parts": [
      [
        2016,
        11,
        28
      ]
    ],
    "date-time": "2016-11-28T15:30:47Z",
    "timestamp": 1480347047000
  },
  "page": "183-188",
  "update-policy": "http://dx.doi.org/10.1016/elsevier_cm_policy",
  "source": "Crossref",
  "is-referenced-by-count": 59,
  "special_numbering": "C",
  "title": [
    "Exoskeleton plantarflexion assistance for elderly"
  ],
  "prefix": "10.1016",
  "volume": "52",
  "author": [
    {
      "given": "S.",
      "family": "Galle",
      "sequence": "first",
      "affiliation": []
    },
    {
      "given": "W.",
      "family": "Derave",
      "sequence": "additional",
      "affiliation": []
    },
    {
      "given": "F.",
      "family": "Bossuyt",
      "sequence": "additional",
      "affiliation": []
    },
    {
      "given": "P.",
      "family": "Calders",
      "sequence": "additional",
      "affiliation": []
    },
    {
      "given": "P.",
      "family": "Malcolm",
      "sequence": "additional",
      "affiliation": []
    },
    {
      "given": "D.",
      "family": "De Clercq",
      "sequence": "additional",
      "affiliation": []
    }
  ],
  "member": "78",
  "reference": [
    {
      "key": "10.1016/j.gaitpost.2016.11.040_bib0005",
      "doi-asserted-by": "crossref",
      "first-page": "212",
      "DOI": "10.1038/nature14288",
      "article-title": "Reducing the energy cost of human walking using an unpowered exoskeleton",
      "volume": "522",
      "author": "Collins",
      "year": "2015",
      "journal-title": "Nature"
    },
    {
      "key": "10.1016/j.gaitpost.2016.11.040_bib0010",
      "doi-asserted-by": "crossref",
      "first-page": "e56137",
      "DOI": "10.1371/journal.pone.0056137",
      "article-title": "A simple exoskeleton that assists plantarflexion can reduce the metabolic cost of human walking",
      "volume": "8",
      "author": "Malcolm",
      "year": "2013",
      "journal-title": "PLoS One"
    },
    {
      "key": "10.1016/j.gaitpost.2016.11.040_bib0015",
      "doi-asserted-by": "crossref",
      "first-page": "1",
      "DOI": "10.1186/s12984-016-0111-3",
      "article-title": "Biomechanical walking mechanisms underlying the metabolic reduction caused by an autonomous exoskeleton",
      "volume": "13",
      "author": "Mooney",
      "year": "2016",
      "journal-title": "J. Neuroeng. Rehabil."
    },
    {
      "key": "10.1016/j.gaitpost.2016.11.040_bib0020",
      "doi-asserted-by": "crossref",
      "first-page": "2341",
      "DOI": "10.1007/s00421-014-2955-1",
      "article-title": "Enhancing performance during inclined loaded walking with a powered ankle-foot exoskeleton",
      "volume": "114",
      "author": "Galle",
      "year": "2014",
      "journal-title": "Eur. J. Appl. Physiol."
    },
    {
      "key": "10.1016/j.gaitpost.2016.11.040_bib0025",
      "doi-asserted-by": "crossref",
      "first-page": "620",
      "DOI": "10.1016/j.gaitpost.2006.07.002",
      "article-title": "Effect of augmented plantarflexion power on preferred walking speed and economy in young and older adults",
      "volume": "25",
      "author": "Norris",
      "year": "2007",
      "journal-title": "Gait Posture"
    },
    {
      "key": "10.1016/j.gaitpost.2016.11.040_bib0030",
      "doi-asserted-by": "crossref",
      "first-page": "1",
      "DOI": "10.1186/s12984-015-0015-7",
      "article-title": "A neuromechanics-based powered ankle exoskeleton to assist walking post-stroke: a feasibility study",
      "volume": "12",
      "author": "Takahashi",
      "year": "2015",
      "journal-title": "J. Neuroeng. Rehabil."
    },
    {
      "key": "10.1016/j.gaitpost.2016.11.040_bib0035",
      "article-title": "Controlling negative and positive power at the ankle with a soft exosuit",
      "author": "Lee",
      "year": "2016",
      "journal-title": "IEEE Int. Conf. Robot. Autom."
    },
    {
      "key": "10.1016/j.gaitpost.2016.11.040_bib0040",
      "doi-asserted-by": "crossref",
      "first-page": "200",
      "DOI": "10.1152/jappl.1992.73.1.200",
      "article-title": "Effects of age and physical activity status on the speed-aerobic demand relationship of walking",
      "volume": "73",
      "author": "Martin",
      "year": "1992",
      "journal-title": "J. Appl. Physiol."
    },
    {
      "key": "10.1016/j.gaitpost.2016.11.040_bib0045",
      "doi-asserted-by": "crossref",
      "first-page": "877",
      "DOI": "10.2165/00007256-200333120-00002",
      "article-title": "Rate and mechanism of maximal oxygen consumption decline with aging: implications for exercise training",
      "volume": "33",
      "author": "Hawkins",
      "year": "2003",
      "journal-title": "Sport Med."
    },
    {
      "key": "10.1016/j.gaitpost.2016.11.040_bib0050",
      "doi-asserted-by": "crossref",
      "first-page": "549",
      "DOI": "10.2147/CIA.S44112",
      "article-title": "Age-related decrease in physical activity and functional fitness among elderly men and women",
      "volume": "8",
      "author": "Milanović",
      "year": "2013",
      "journal-title": "Clin. Interv. Aging"
    },
    {
      "key": "10.1016/j.gaitpost.2016.11.040_bib0055",
      "doi-asserted-by": "crossref",
      "first-page": "205",
      "DOI": "10.1001/jama.1996.03540030039029",
      "article-title": "Influences of cardiorespiratory fitness and other precursors on cardiovascular disease and all-cause mortality in men and women",
      "volume": "276",
      "author": "Blair",
      "year": "1996",
      "journal-title": "JAMA"
    },
    {
      "key": "10.1016/j.gaitpost.2016.11.040_bib0060",
      "doi-asserted-by": "crossref",
      "first-page": "2004",
      "DOI": "10.1152/jappl.1991.71.5.2004",
      "article-title": "Effects of gender, age, and fitness level on response of VO2max to training in 60–71 yr olds",
      "volume": "71",
      "author": "Kohrt",
      "year": "1991",
      "journal-title": "J. Appl. Physiol."
    },
    {
      "key": "10.1016/j.gaitpost.2016.11.040_bib0065",
      "doi-asserted-by": "crossref",
      "first-page": "S329",
      "DOI": "10.1111/j.1532-5415.2010.02913.x",
      "article-title": "The energetic pathway to mobility loss: an emerging new framework for longitudinal studies on aging",
      "volume": "58",
      "author": "Schrack",
      "year": "2010",
      "journal-title": "J. Am. Geriatr. Soc."
    },
    {
      "key": "10.1016/j.gaitpost.2016.11.040_bib0070",
      "doi-asserted-by": "crossref",
      "first-page": "931",
      "DOI": "10.1097/PHM.0b013e318269d9bb",
      "article-title": "The electrically assisted bicycle: an alternative way to promote physical activity",
      "volume": "91",
      "author": "Louis",
      "year": "2012",
      "journal-title": "Am. J. Phys. Med. Rehabil."
    },
    {
      "key": "10.1016/j.gaitpost.2016.11.040_bib0075",
      "article-title": "Reducing the metabolic cost of walking with an ankle exoskeleton: interaction between actuation timing and power",
      "author": "Galle",
      "year": "2016",
      "journal-title": "J. Neuroeng. Rehabil."
    },
    {
      "key": "10.1016/j.gaitpost.2016.11.040_bib0080",
      "doi-asserted-by": "crossref",
      "first-page": "340",
      "DOI": "10.1093/ptj/70.6.340",
      "article-title": "Biomechanical walking pattern changes in the fit and healthy elderly",
      "volume": "70",
      "author": "Winter",
      "year": "1990",
      "journal-title": "Phys. Ther."
    },
    {
      "key": "10.1016/j.gaitpost.2016.11.040_bib0085",
      "doi-asserted-by": "crossref",
      "first-page": "109",
      "DOI": "10.1186/1743-0003-11-109",
      "article-title": "Combined effects of age and gender on gait symmetry and regularity assessed by autocorrelation of trunk acceleration",
      "volume": "11",
      "author": "Kobayashi",
      "year": "2014",
      "journal-title": "J. Neuroeng. Rehabil."
    },
    {
      "key": "10.1016/j.gaitpost.2016.11.040_bib0090",
      "doi-asserted-by": "crossref",
      "first-page": "21",
      "DOI": "10.1186/1479-5868-6-21",
      "article-title": "The international prevalence study on physical activity: results from 20 countries",
      "volume": "6",
      "author": "Bauman",
      "year": "2009",
      "journal-title": "Int. J. Behav. Nutr. Phys. Act."
    },
    {
      "key": "10.1016/j.gaitpost.2016.11.040_bib0095",
      "doi-asserted-by": "crossref",
      "first-page": "495",
      "DOI": "10.1016/j.gaitpost.2013.01.029",
      "article-title": "Adaptation to walking with an exoskeleton that assists ankle extension",
      "volume": "38",
      "author": "Galle",
      "year": "2013",
      "journal-title": "Gait Posture"
    },
    {
      "key": "10.1016/j.gaitpost.2016.11.040_bib0100",
      "doi-asserted-by": "crossref",
      "first-page": "14",
      "DOI": "10.1186/s12984-015-0014-8",
      "article-title": "The influence of push-off timing in a robotic ankle-foot prosthesis on the energetics and mechanics of walking",
      "volume": "12",
      "author": "Malcolm",
      "year": "2015",
      "journal-title": "J. Neuroeng. Rehabil."
    },
    {
      "key": "10.1016/j.gaitpost.2016.11.040_bib0105",
      "doi-asserted-by": "crossref",
      "first-page": "266",
      "DOI": "10.2519/jospt.1997.26.5.266",
      "article-title": "Optimal walking in terms of variability in step length",
      "volume": "26",
      "author": "Sekiya",
      "year": "1997",
      "journal-title": "J. Orthop. Sport Phys. Ther."
    },
    {
      "key": "10.1016/j.gaitpost.2016.11.040_bib0110",
      "volume": "vol. 2nd",
      "author": "Cohen",
      "year": "1988"
    },
    {
      "key": "10.1016/j.gaitpost.2016.11.040_bib0115",
      "doi-asserted-by": "crossref",
      "DOI": "10.1177/0003489416629979",
      "article-title": "Balance disorders in the elderly: does instability increase over time?",
      "author": "Soto-Varela",
      "year": "2016",
      "journal-title": "Ann. Otol. Rhinol. Laryngol."
    },
    {
      "key": "10.1016/j.gaitpost.2016.11.040_bib0120",
      "doi-asserted-by": "crossref",
      "first-page": "1050",
      "DOI": "10.1053/apmr.2001.24893",
      "article-title": "Gait variability and fall risk in community-living older adults: a 1-year prospective study",
      "volume": "82",
      "author": "Hausdorff",
      "year": "2001",
      "journal-title": "Arch. Phys. Med. Rehabil."
    },
    {
      "key": "10.1016/j.gaitpost.2016.11.040_bib0125",
      "doi-asserted-by": "crossref",
      "unstructured": "J.A. Norris, A. Marsh, K.P. Granata, S.D. Ross, Positive feedback in powered exoskeletons: Improved metabolic efficiency at the cost of reduced stability? Proc. Asme Int. Des. Eng. Tech. Conf. Comput. Inf. Eng. Conf. 2007, Vol 5, Pts a-C, New York: Amer Soc Mechanical Engineers; 2008, p. 1619–26. 10.1115/DETC2007-35657.",
      "DOI": "10.1115/DETC2007-35657"
    },
    {
      "key": "10.1016/j.gaitpost.2016.11.040_bib0130",
      "doi-asserted-by": "crossref",
      "first-page": "72",
      "DOI": "10.1016/j.gaitpost.2004.01.003",
      "article-title": "Familiarisation to treadmill walking in unimpaired older people",
      "volume": "21",
      "author": "Wass",
      "year": "2005",
      "journal-title": "Gait Posture"
    },
    {
      "key": "10.1016/j.gaitpost.2016.11.040_bib0135",
      "doi-asserted-by": "crossref",
      "first-page": "347",
      "DOI": "10.1164/rccm.201204-0596PP",
      "article-title": "Global strategy for the diagnosis, management, and prevention of chronic obstructive pulmonary disease GOLD executive summary",
      "volume": "187",
      "author": "Vestbo",
      "year": "2013",
      "journal-title": "Am. J. Respir. Crit. Care Med."
    },
    {
      "key": "10.1016/j.gaitpost.2016.11.040_bib0140",
      "doi-asserted-by": "crossref",
      "first-page": "317",
      "DOI": "10.1016/S0003-9993(98)90013-2",
      "article-title": "Biomechanical gait alterations independent of speed in the healthy elderly: evidence for specific limiting impairments",
      "volume": "79",
      "author": "Kerrigan",
      "year": "1998",
      "journal-title": "Arch. Phys. Med. Rehabil."
    },
    {
      "key": "10.1016/j.gaitpost.2016.11.040_bib0145",
      "doi-asserted-by": "crossref",
      "first-page": "246",
      "DOI": "10.1016/j.gaitpost.2014.10.015",
      "article-title": "Uphill walking with a simple exoskeleton: plantarflexion assistance leads to proximal adaptations",
      "volume": "41",
      "author": "Galle",
      "year": "2015",
      "journal-title": "Gait Posture"
    },
    {
      "key": "10.1016/j.gaitpost.2016.11.040_bib0150",
      "doi-asserted-by": "crossref",
      "first-page": "97",
      "DOI": "10.1186/s12984-015-0086-5",
      "article-title": "Learning to walk with an adaptive gain proportional myoelectric controller for a robotic ankle exoskeleton",
      "volume": "12",
      "author": "Koller",
      "year": "2015",
      "journal-title": "J. Neuroeng. Rehabil."
    }
  ],
  "container-title": [
    "Gait &amp; Posture"
  ],
  "language": "en",
  "link": [
    {
      "URL": "https://api.elsevier.com/content/article/PII:S0966636216306798?httpAccept=text/xml",
      "content-type": "text/xml",
      "content-version": "vor",
      "intended-application": "text-mining"
    },
    {
      "URL": "https://api.elsevier.com/content/article/PII:S0966636216306798?httpAccept=text/plain",
      "content-type": "text/plain",
      "content-version": "vor",
      "intended-application": "text-mining"
    }
  ],
  "deposited": {
    "date-parts": [
      [
        2018,
        9,
        6
      ]
    ],
    "date-time": "2018-09-06T05:54:41Z",
    "timestamp": 1536213281000
  },
  "score": 0.0,
  "resource": {
    "primary": {
      "URL": "https://linkinghub.elsevier.com/retrieve/pii/S0966636216306798"
    }
  },
  "issued": {
    "date-parts": [
      [
        2017,
        2
      ]
    ]
  },
  "references-count": 30,
  "alternative-id": [
    "S0966636216306798"
  ],
  "URL": "https://doi.org/10.1016/j.gaitpost.2016.11.040",
  "ISSN": [
    "0966-6362"
  ],
  "issn-type": [
    {
      "value": "0966-6362",
      "type": "print"
    }
  ],
  "published": {
    "date-parts": [
      [
        2017,
        2
      ]
    ]
  },
  "assertion": [
    {
      "value": "Elsevier",
      "name": "publisher",
      "label": "This article is maintained by"
    },
    {
      "value": "Exoskeleton plantarflexion assistance for elderly",
      "name": "articletitle",
      "label": "Article Title"
    },
    {
      "value": "Gait & Posture",
      "name": "journaltitle",
      "label": "Journal Title"
    },
    {
      "value": "https://doi.org/10.1016/j.gaitpost.2016.11.040",
      "name": "articlelink",
      "label": "CrossRef DOI link to publisher maintained version"
    },
    {
      "value": "article",
      "name": "content_type",
      "label": "Content Type"
    },
    {
      "value": "© 2016 Elsevier B.V. All rights reserved.",
      "name": "copyright",
      "label": "Copyright"
    }
  ]
}
