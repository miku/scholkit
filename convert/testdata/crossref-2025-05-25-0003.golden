{
    "source": "crossref",
    "contribs": [
        {
            "extra": {},
            "given_name": "S.",
            "raw_name": "S. Galle",
            "role": "author",
            "surname": "Galle"
        },
        {
            "extra": {},
            "given_name": "W.",
            "raw_name": "W. Derave",
            "role": "author",
            "surname": "Derave"
        },
        {
            "extra": {},
            "given_name": "F.",
            "raw_name": "F. Bossuyt",
            "role": "author",
            "surname": "Bossuyt"
        },
        {
            "extra": {},
            "given_name": "P.",
            "raw_name": "P. Calders",
            "role": "author",
            "surname": "Calders"
        },
        {
            "extra": {},
            "given_name": "P.",
            "raw_name": "P. Malcolm",
            "role": "author",
            "surname": "Malcolm"
        },
        {
            "extra": {},
            "given_name": "D.",
            "raw_name": "D. De Clercq",
            "role": "author",
            "surname": "De Clercq"
        }
    ],
    "ext_ids": {
        "doi": "10.1016/j.gaitpost.2016.11.040"
    },
    "extra": {
        "arxiv": {},
        "dblp": {},
        "openalex": {
            "open_access": {}
        },
        "crossref": {},
        "oai": {}
    },
    "id": "crossref-8edd99f181232f2a704904d6f1fe3bfdb1184cb5",
    "release_date": "48880-04-30",
    "release_type": "article-journal",
    "title": "Exoskeleton plantarflexion assistance for elderly"
}